config/prow/plugins.yaml

```diff
@@ -767,13 +767,13 @@ require_matching_label:
   org: kubernetes-sigs
   repo: cluster-api-provider-aws
   issues: true
-  prs: false
+  prs: true
```

Comment:
Was that intentional?

Reply:
Yes, we want triage bot in the issues itself.